EpikCord/__init__.py

```diff
@@ -696,6 +699,11 @@ async def wrapper():
                 await self.heartbeat(True)
 
             elif event["op"] == GatewayOpcode.HEARTBEAT_ACK:
+                heartbeat_ack_time = time_ns()
```

Comment:
`perf_counter_ns` is more suitable for timers

Reply:
Ok I will commit tmrw